operations/services.md

```diff
@@ -1579,6 +1579,314 @@ Returns all companionships based on customers, reservations or reservation group
+    "ServiceIds": [
+        "bd26d8db-86da-4f96-9efc-e5a4654a4a94"
+    ],
+    "ResourceAccessTokenIds": [
+        "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D"
+    ],
+    "ReservationIds ": [
+        "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D"
+    ],
+    "CollidingUtc": {
+        "StartUtc": "2020-02-15T00:00:00Z",
+        "EndUtc": "2020-02-20T00:00:00Z"
+    },
+    "ActivityStates": [
+        "Active"
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Service](#service)s where the resource access tokens belong to. |
+| `ResourceAccessTokenIds` | array of string | optional, max 1000 items | Unique identifiers of [Resource access token](#resource-access-token)s. Required if no other filter is provided. |
+| `ReservationIds` | array of string | optional, max 1000 items | Unique identifiers of [Reservation](reservations.md#reservation)s. Required if no other filter is provided. |
+| `CollidingUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Resource access token](#resource-access-token) is active. Required if no other filter is provided. |
+| `ActivityStates` | array of string [Activity state](#activity-state) | required | Whether return only active, only deleted or both records. |
+
+### Response
+
+```javascript
+{
+    "ResourceAccessTokens": [
+        {
+            "Id": "72d4b117-1f84-44a3-1f84-8b2c0635ac60",
+            "ServiceOrderId": "35d4b117-4e60-44a3-9580-c582117eff98",
+            "CompanionshipId": "25d4b117-4e60-44a3-9580-c582117eff98",
+            "ResourceId": "65d4b117-4e60-44a3-9580-c582117eff98",
+            "Type": "PinCode",
+            "Value": "resource access token value",
+            "SerialNumber": null,
+            "ValidityStartUtc": "2020-10-09T22:00:00Z",
+            "ValidityEndUtc": "2020-10-10T22:00:00Z",
+            "Permissions":
+            {
+                "Bed": true,
+                "Room": false,
+                "Floor": false,
+                "Building": false
+            },
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ResourceAccessTokens` | array of [Resource access token](#resource-access-token) | required | Resource access tokens. |
+
+#### Resource access token
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of [Resource access token](#resource-access-token). |
+| `ServiceOrderId` | string | required | Unique identifier of [Reservation](reservations.md#reservation). |
```

Comment:
I' using `identifier od a service order (for example Reservation)` to be future proof.